src/Grpc.AspNetCore.Server/Internal/HttpContextServerCallContext.cs

```diff
@@ -257,9 +259,25 @@ private void EndCallCore()
                 HttpContext.Response.ConsolidateTrailers(this);
             }
 
+            LogCallEnd();
+
             _callComplete = true;
         }
 
+        private void LogCallEnd()
+        {
+            var currentActivity = Activity.Current;
+            if (currentActivity != null)
+            {
+                currentActivity.AddTag("GrpcStatus", _status.StatusCode.ToTrailerString());
```

Comment:
I suggest `grpc.status_code` similarly to `http.status_code` which is commonly used in OpenCeneus/Tracing/Telemetry as status tag on HTTP